lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/ZipLoader.java

```diff
@@ -58,6 +58,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import org.apache.commons.io.FilenameUtils;
```

Comment:
Yeah, I didn't notice that but I think since there's two different licenses in here I'm not sure what to do with it and better just to leave it for another cleanup.